contrib/Shopify/sarama/option.go

```diff
@@ -54,6 +56,8 @@ func WithServiceName(name string) Option {
 }
 
 // WithDataStreams enables the Data Streams monitoring product features: https://www.datadoghq.com/product/data-streams-monitoring/
+//
+// Deprecated: Use the environment variable DD_DATA_STREAMS_ENABLED instead
```

Comment:
why can't we keep both? I think its a common pattern in the Go tracer to provide both in-code and env variable choices for configurations

Reply:
hmm, yes that's an argument that can be heard. Though it lets go users do stuff that won't work in other tracers, which is not _the best_, but being consistent within the go tracer itself is probably more important.